modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -408,18 +408,14 @@ function DOMUtils(doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
     }
 
     // Make a HTML5 safe shallow copy
-    if (!deep) {
-      const clone = doc.createElement(node.nodeName);
+    const clone = doc.createElement(node.nodeName);
```

Comment:
could we nest this inside an `else` instead of flattening it 🤔

Reply:
Done.